### DISCUSSION TITLE
Getting stated contribute

### DISCUSSION THREAD

**Discussion by AxelGard** [fastapi/fastapi#5431] [Questions] [answered]
I want to start contributing to the FastAPI project. 

What should I know?? 
I have looked over the https://fastapi.tiangolo.com/contributing/ 

Is there some module that one should take an extra good look at to understand the project?? 

How hard is there to get things merged, since there are 400 open PR. 
Is it hard to get things merged?? 

**Thanks for any answers!!**

**Comment by iudeen** [comment, marked as answer]
You can start as is. You should be familiar with ASGI spec, the Starlette library, Pydantic to start with. Then knowledge about web frameworks also helps.

The maintainer of the project reviews most PR before each release and majority of the open PRs are usually for docs that are yet to be fully completed.

**Reply by AxelGard** [reply]
Okay, good to know! 
Is the discussion page the official communication channel? 

Thanks for the answer!  :smile:

**Reply by iudeen** [reply]
We have a [discord server](https://discord.gg/VQjSZaeJmf) as well.

**Reply by tiangolo** [reply]
Also, the best help I can get is actually carefully reviewing PRs and helping with questions in issues. There's a lot of new details here including other ways to help: https://fastapi.tiangolo.com/help-fastapi/
